classes/rental.rb

```diff
@@ -0,0 +1,15 @@
+# Define class rentals
+class Rental
+  attr_reader :date
```

Comment:
- [ ]  Good job for creating the attributes for Rental class. One thing to be improved here is that date attribute should have both `setter `and `getter`.  When you use `attr_reader `on date, you limit it to be changed or updated . fix it for your app to work properly.